cfg/config.go

```diff
@@ -448,6 +448,20 @@ func ParseField(v reflect.Value, vField reflect.Value, tField reflect.StructFiel
 			}
 			finalField.SetInt(int64(value))
 
+		case "unit":
+			value, err := strconv.Atoi(strings.Split(vField.String(), " ")[0])
+			if err != nil {
+				return fmt.Errorf("could not parse number %d, err: %s", value, err.Error())
```

Comment:
err: %w?

Reply:
Error handling is done as in other cases